.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 120
+ignore = D100, D103, D104, D401, E203, E501, E731, E741, F401, W503
+docstring-convention = numpy
```

Comment:
also what docstring convention is this? I usually use the Google python style -- will it be ok/get auto-reformatted?

Reply:
I'm OK switching to the google convention - it doesn't really matter to me, I just got used to using the numpy one which has slightly different style.